tests/FakeItEasy.Tests/Creation/CastleDynamicProxy/CastleDynamicProxyGeneratorTests.cs

```diff
@@ -207,7 +212,11 @@ public void Should_specify_that_no_default_constructor_was_found()
             result.ReasonForFailure.Should().StartWith("No usable default constructor was found on the type");
         }
 
+#if !NETCORE
         [Fact]
+#else
+        [Fact(Skip = "System.AppDomainInitializerInfo is not available in .NET Core. Furthermore, even if we could found another non-static internal class in mscorlib that is type-forwarded to Private.CoreLib on .NET Core it may disappear in the future too.")]

```

Comment:
Did you get a chance to test on .NET Core?  It works on desktop, but failed on .NET Core, with the following call returning null.  The [FluentAssertions repo seems to indicate that it's only included in NET40 and NET45 projects.](https://github.com/dennisdoomen/FluentAssertions/search?utf8=%E2%9C%93&q=AppSettingsConfigurationStore)

```
    var type = Type.GetType("FluentAssertions.Common.AppSettingsConfigurationStore, FluentAssertions");
```